File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/VideoFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.std.files.Video;

public class VideoFilterValidator extends FilterValidator<Video> {

    public VideoFilterValidator(CustomValidator<Video> validator) {
        super(Video.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/VideoFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.telegram.bot.api.model.objects.std.files.Video
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class VideoFilterValidator(validator: Validator<Video>) : FilterValidator<Video>(Video::class, validator)

